shanoir-ng-datasets/src/main/java/org/shanoir/ng/dataset/controler/DatasetApiController.java

```diff
@@ -610,10 +621,23 @@ public ResponseEntity<ByteArrayResource> downloadStatistics(
 			@Parameter(description = "Subject name excluding regular expression", required=false) @Valid
 			@RequestParam(value = "subjectNameOutRegExp", required = false) String subjectNameOutRegExp
 			) throws RestServiceException, IOException {
+
+		ShanoirEvent event = null;
+		float progress = 0;
+		event = new ShanoirEvent(
+				ShanoirEventType.DOWNLOAD_STATISTICS_EVENT,
+				null,
+				KeycloakUtil.getTokenUserId(),
+				"Fetching statistics...",
+				ShanoirEvent.IN_PROGRESS,
+				null);
+
+		eventService.publishEvent(event);
+
```

Comment:
I think the idea here was to maje an @Async method so that we don't have to wait for the method ending.
-> We click on download stats
-> A popup appears saying "statistics successfully started"
-> A job appears on the same time with a progress
-> At the end of the progress, we can download the job